desktop/release-notes.md

```diff
@@ -23,17 +23,79 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
-> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.9.1-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
-> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.9.1-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
+> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.10.0-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.10.0-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
+> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.10.0-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
+
+### Upgrades
+- [Docker Engine to v20.10.17](https://docs.docker.com/engine/release-notes/#201017)
+- [Docker Compose v2.6.1](https://github.com/docker/compose/releases/tag/v2.6.1)
+- [Kubernetes v1.41.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.41.1)
+- [cri-dockerd to v0.2.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.1)
+- [CNI plugins to v1.1.1](https://github.com/containernetworking/releases/tag/v1.1.1)
+- [containerd to v1.6.6](https://github.com/containerd/containerd/releases/tag/v1.6.6)
+- [runc to v1.1.2](https://github.com/opencontainers/runc/releases/tag/v1.1.2)
+- [Go 1.18.3](https://github.com/golang/go/releases/tag/go1.18.3)
+
+### New
+
+- TODO: Waiting for highlights.
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Containers tab: Bulk actions for start/pause/stop selected containers.
+- Containers tab: Pause/restart actions for compose projects.
+- External URLs can refer to extension details in Docker Desktop Marketplace using links like `docker-desktop://extensions/marketplace?extensionId=docker/logs-explorer-extension`.
+- Implemented changes on the logs search functionality and added the possibility of clearing the terminal.
+- Expanded / collapsed state of the Compose apps is now persisted.
+- `docker extension` CLI commands are available with Docker Desktop by default.
+- Extensions: make screenshots displayed in the marketplace bigger.
+- Fixed a bug where a Docker extension fails to load if its backend container(s) are stopped. Fixes [docker/extensions-sdk#16](https://github.com/docker/extensions-sdk/issues/162).
+- Fixed a bug where license agreement is not showing up, thus silently blocking Docker Desktop startup.
+- Fixed the displayed image and tag for unpublished extensions to actually display the ones from the installed unpublished extension.
+- Fixed the duplicate footer on the Support screen.
```

Comment:
From the Environments team, I'd like to add a line for the work done in https://github.com/docker/dev-envs/pull/186, which addresses https://github.com/docker/dev-environments/issues/16

```suggestion
- Fixed the duplicate footer on the Support screen.
- Dev Environments can be created from a subdirectory in a GitHub repository
```